cmd/osv-scanner/testmain_test.go

```diff
@@ -1,14 +1,40 @@
 package main
 
 import (
+	"context"
+	"io"
+	"log"
+	"log/slog"
 	"os"
+	"strings"
 	"testing"
 
 	"github.com/go-git/go-git/v5"
 	"github.com/google/osv-scanner/v2/internal/testutility"
 )
 
+// muffledHandler eats certain log messages to reduce noise in the test output
+type muffledHandler struct {
+	slog.TextHandler
+}
+
+func (c *muffledHandler) Handle(ctx context.Context, record slog.Record) error {
+	// todo: work with the osv-scalibr team to see if we can reduce these
+	if strings.HasPrefix(record.Message, "Neither CPE nor PURL found for package") ||
```

Comment:
Can you add the logging output that gets spammed when doing container scanning as well. 
e.g.
```
2025/02/19 15:37:54 Starting filesystem walk for root: 
2025/02/19 15:37:54 End status: 0 dirs visited, 1 inodes visited, 1 Extract calls, 154.071µs elapsed, 154.121µs wall time
```